astropy/table/jsviewer.py

```diff
@@ -23,7 +23,7 @@ class Conf(_config.ConfigNamespace):
         'The URL to the jquery datatables library.')
 
     css_urls = _config.ConfigItem(
-        ['https://cdn.datatables.net/1.10.9/css/jquery.dataTables.min.css'],
+        ['https://cdn.datatables.net/1.10.9/css/jquery.dataTables.css'],
```

Comment:
I thought we were using the minified ones? Why change the default? (and same comment below)